app/helpers/alchemy/elements_helper.rb

```diff
@@ -224,14 +229,17 @@ def element_tags_attributes(element, options = {})
     #
     # @param [Array] elements - The elements you want to sort
     # @param [String] content_name - The name of the content you want to sort by
+    # @param [Boolean] reverse - Reverse the sorted elements order
     #
     # @return [Array]
     #
-    def sort_elements_by_content(elements, content_name)
-      elements.sort_by do |element|
+    def sort_elements_by_content(elements, content_name, reverse = false)
+      sorted_elements = elements.sort_by do |element|
```

Comment:
Style/BlockDelimiters: Prefer `{...}` over `do...end` for functional blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)